blog/admin.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.models import Group
+
+from django.contrib import admin
+from blog.models import Post, User, Commentary
+
+
+@admin.register(User)
+class UserAdmin(UserAdmin):
+    list_display = ("is_staff", "is_active")
+    search_fields = ("username", "email")
+
+
+@admin.register(Post)
+class PostAdmin(admin.ModelAdmin):
+    list_display = ("created_time", "owner")
```

Comment:
The 'list_display' attribute for 'PostAdmin' should include fields that help identify the post, such as 'title', to provide more meaningful information in the admin interface.